strings/names/japanese_prefix.txt

```diff
@@ -0,0 +1,16 @@
+Anime
+Girugamesh
+Tokyo
+Gaybang
```

Comment:
uuuuuuuh, don't think that it's a good idea to add

Reply:
This was my idea I love the 2009 sakuracon video. I do understand doe if its gotta go